src/app/page/UserProfile/LinkPhoneAccount.jsx

```diff
@@ -32,7 +32,7 @@ function LinkPhoneAccount({ firebase, auth, data, errorHandler }) {
       await confirmationResult.confirm(verificationCode);
       verifier.clear();
       setSuccessSnackbarOpen(true);
-      window.location.reload(false);
+      //window.location.reload(false);
```

Comment:
So the idea was to refresh the page, so that the User Profile page reflects the change. Can we do a force refresh instead of commenting it out?